pkg/kubelet/network/cni/cni.go

```diff
@@ -43,6 +44,8 @@ type cniNetworkPlugin struct {
 	host           network.Host
 	execer         utilexec.Interface
 	nsenterPath    string
+	mu             sync.Mutex
+	podMutexes     map[string]*sync.Mutex
```

Comment:
@yujuhong it doesn't currently cause any problems because GetPodStatus() errors are actually ignored.

I agree with @thockin that this should probably be fixed in the runtimes and documented that the runtime should call network plugin operations sequentially for a given pod.  So I'll close this.